src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Box, ExpandableLabel } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+import usePersistState from 'hooks/usePersistState'
+
+const Wrapper = styled(Box)<{ isVisible: boolean }>`
+  display: flex;
+  flex-direction: column;
+  justify-content: ${({ isVisible }) => (isVisible ? 'space-between' : 'flex-end')};
+  background: #452a7a;
+  box-shadow: inset 0px -4px 4px rgba(0, 0, 0, 0.25);
+  padding: ${({ isVisible }) => (isVisible ? '32px' : '0 32px')};
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-direction: row;
+  }
+`
+
+const Content = styled.div<{ isVisible: boolean }>`
+  display: ${({ isVisible }) => (isVisible ? 'block' : 'none')};
+`
+
+const Banner: React.FC<{ id: string }> = ({ id, children, ...props }) => {
+  const [isVisible, setIsVisible] = usePersistState(true, `banner-${id}`)
+  return (
+    <Wrapper isVisible={isVisible} {...props}>
```

Comment:
Totally depends on how much time you have to give to this, but might be useful to have the option to render some content when the banner is collapsed? In this instance it would maybe be helpful to users to show:

```
<Text color="white" fontSize="24px" bold>
 ACTION REQUIRED for all LP token holders
</Text>
```

Reply:
yes, will do